containerserver/server.go

```diff
@@ -272,7 +272,11 @@ func (server *ContainerServer) ContainerPutHandler(writer http.ResponseWriter, r
 	}
 	created, db, err := server.containerEngine.Create(vars, timestamp, metadata, policyIndex, defaultPolicyIndex)
 	if err == ErrorPolicyConflict {
-		srv.StandardResponse(writer, http.StatusConflict)
+		if policyIndex == -1 {
```

Comment:
Can this happen?  It looks like ErrorPolicyConflict is only returned when policyIndex >= 0.